src/sdkV2.ts

```diff
@@ -425,6 +451,7 @@ export default class CegaEvmSDKV2 {
     quoteAsset: EvmAddress,
     timestamp: number,
     oracleDataSource: OracleDataSourceDcs,
+    overrides: TxOverrides = {},
```

Comment:
you don't need overrides for this. `getPrice` should be a view function

Reply:
Yep, forgot to remove the arg in the method